guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
<p>Can we remove "one of the best", or qualify it with "one of the best one-stage object detection models". According to <a href="https://paperswithcode.com/sota/object-detection-on-coco" target="_blank">https://paperswithcode.com/sota/object-detection-on-coco</a>, RetinaNet is now ranked 50+ in object detection.</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>